{"uid":"8e2883c8afe5526c","name":"Oops...we found an error preparing this test file: cypress/e2e/Topoc33_02.cy.js","fullName":"Oops...we found an error preparing this test file: cypress/e2e/Topoc33_02.cy.js","time":{"start":1685268695098,"stop":1685268695098,"duration":0},"status":"broken","statusMessage":"Oops...we found an error preparing this test file:\n\n  > cypress\\e2e\\Topoc33_02.cy.js\n\n","statusTrace":"\n\nError: Build failed with 668 errors:\nnode_modules/@azure/core-rest-pipeline/dist/index.js:7:17: ERROR: Could not resolve \"os\"\nnode_modules/@azure/core-rest-pipeline/dist/index.js:13:19: ERROR: Could not resolve \"util\"\nnode_modules/@azure/core-rest-pipeline/dist/index.js:14:19: ERROR: Could not resolve \"http\"\nnode_modules/@azure/core-rest-pipeline/dist/index.js:15:20: ERROR: Could not resolve \"https\"\nnode_modules/@azure/core-rest-pipeline/dist/index.js:16:19: ERROR: Could not resolve \"zlib\"\n...\n    at failureErrorWithLog (D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:1636:15)\n    at D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:1048:25\n    at runOnEndCallbacks (D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:1471:45)\n    at buildResponseToResult (D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:1046:7)\n    at D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:1075:16\n    at responseCallbacks.<computed> (D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:697:9)\n    at handleIncomingPacket (D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:752:9)\n    at Socket.readFromStdout (D:\\cyprus Automation\\Adv Cypress\\node_modules\\esbuild\\lib\\main.js:673:7)\n    at Socket.emit (node:events:513:28)\n    at addChunk (node:internal/streams/readable:324:12)\n    at readableAddChunk (node:internal/streams/readable:297:9)\n    at Readable.push (node:internal/streams/readable:234:10)\n    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23)\n\nThis occurred while Cypress was compiling and bundling your test code. This is usually caused by:\n\n- A missing file or dependency\n- A syntax error in the file or one of its dependencies\n\nFix the error in your code and re-run your tests.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"cypress/e2e/Topoc33_02.cy.js"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"8e2883c8afe5526c.json","parameterValues":[]}